packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -100,20 +104,20 @@ class BillingClient {
   /// This triggers the creation of a new `BillingClient` instance in Java if
   /// one doesn't already exist.
   Future<BillingResultWrapper> startConnection(
-      {@required
-          OnBillingServiceDisconnected onBillingServiceDisconnected}) async {
+      {required OnBillingServiceDisconnected
+          onBillingServiceDisconnected}) async {
     assert(_enablePendingPurchases,
         'enablePendingPurchases() must be called before calling startConnection');
     List<Function> disconnectCallbacks =
         _callbacks[_kOnBillingServiceDisconnected] ??= [];
     disconnectCallbacks.add(onBillingServiceDisconnected);
-    return BillingResultWrapper.fromJson(await channel
+    return BillingResultWrapper.fromJson((await channel
         .invokeMapMethod<String, dynamic>(
             "BillingClient#startConnection(BillingClientStateListener)",
             <String, dynamic>{
           'handle': disconnectCallbacks.length - 1,
           'enablePendingPurchases': _enablePendingPurchases
-        }));
+        }))!);
```

Comment:
Are we positive there are no codepaths in any of the native code that return null for all these calls you're asserting?